templates/docker/base.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+def source_paths
+  [__dir__]
+end
+
+copy_file(
+  "files/Dockerfile",
+  "Dockerfile"
```

Comment:
Is it possible to have a template right in this file instead of copying and gsubing some variables?

Just like in https://github.com/DmitryBarskov/rails-template/blob/main/templates/rubocop.rb#L24